src/main/java/vg/civcraft/mc/civmodcore/dao/ManagedDatasource.java

```diff
@@ -62,7 +66,8 @@
 	private static final long MAX_WAIT_FOR_LOCK = 600000l;
 	private static final long WAIT_PERIOD = 500l;
 	
-	private ConnectionPool connections;
+	private HikariDataSource connections;
```

Comment:
Yes, I made this in a rush last night to get reactions on it, that was my first thought too.